packages/mobile/src/screens/chat-screen/ChatScreen.tsx

```diff
@@ -190,7 +202,7 @@ export const ChatScreen = () => {
   const chatFrozenRef = useRef(chat)
 
   useEffect(() => {
-    if (chatId && (chat?.messagesStatus ?? Status.IDLE) === Status.IDLE) {
+    if (chatId && statusIsNotFinalized(chat?.messagesStatus ?? Status.IDLE)) {
```

Comment:
I don't think this is what we want. We only want to start fetching if we're not currently fetching - eg. iff `Status.IDLE` but not `Status.LOADING`

TIL we have this helper though! that's neat

Reply:
good looks